src/handlers/utils.js

```diff
@@ -62,20 +67,86 @@ export function createAskArticleSubmissionReply(issuedAt) {
     '\n' +
     '雖然您不會立刻收到查證結果，但可以幫助到未來同樣收到這份訊息的人。\n' +
     '\n' +
-    '請按左下角「⌨️」鈕，把「為何您會覺得這是一則謠言」的理由傳給我們，幫助闢謠編輯釐清您有疑惑之處。';
+    '請按左下角「⌨️」鈕，把「為何您會覺得這是一則謠言」的理由傳給我們，幫助闢謠編輯釐清您有疑惑之處。' +
+    '\n' +
+    '若想放棄，請輸入「n」。';
+  let accountName = process.env.LINE_AT_NAME || 'cofacts';
```

Comment:
Should be `LINE_AT_ID` and `const`

Reply:
OMG I missed this. Sorry about messing this up.... I guess it's okay that I reset my commit and force push it to the branch since I'm the only one working on it?